repository-data/application/src/main/resources/hcm-config/namespaces/hee/tabPanel.yaml

```diff
@@ -62,13 +69,25 @@ definitions:
               jcr:primaryType: frontend:pluginconfig
           /tabBody:
             jcr:primaryType: frontend:plugin
-            caption: Tab body
+            caption: Tab content
             field: tabBody
             hint: ''
             plugin.class: org.hippoecm.frontend.editor.plugins.field.NodeFieldPlugin
             wicket.id: ${cluster.id}.field
             /cluster.options:
               jcr:primaryType: frontend:pluginconfig
-              ckeditor.config.appended.json: '{ stylesSet: "styles_without_h1:./styles_without_h1.js"
+              ckeditor.config.appended.json: '{ removeButtons:"Table",stylesSet: "styles_without_h1:./styles_without_h1.js"
                 }'
               ckeditor.config.overlayed.json: '{ forcePasteAsPlainText: true }'
+          /contentBlocks:
+            jcr:primaryType: frontend:plugin
+            caption: Tab content objects
+            compoundList: hee:ActionLink,hippostd:html,hee:richTextReference
```

Comment:
Per spec, the content block should include `hee:expanderTableReference` block as well.